examples/lock-app/cyw30739/src/ZclCallbacks.cpp

```diff
@@ -61,3 +62,94 @@ void MatterPostAttributeChangeCallback(const app::ConcreteAttributePath & attrib
 
     printf("ERROR clusterId: 0x%04lx, unknown attribute ID: 0x%04lx\n", attributePath.mClusterId, attributePath.mAttributeId);
 }
+
+/** @brief DoorLock Cluster Init
+ *
+ * This function is called when a specific cluster is initialized. It gives the
+ * application an opportunity to take care of cluster initialization procedures.
+ * It is called exactly once for each endpoint where cluster is present.
+ *
+ * @param endpoint   Ver.: always
```

Comment:
what does `Ver.: always` mean?

I can read the code as `this takes a single parameter representing the endpoint id` ... the comment confuses me more than it helps.